Runtime/Scripts/RectTransformFilterHandler.cs

```diff
@@ -0,0 +1,43 @@
+﻿using UnityEngine;
+using BetaHub;
+
+public class RectTransformFilterHandler : MonoBehaviour
+{
+    public GameRecorder gameRecorder;
+    public string keyFilter = "box"; // Filter key
+    public Color boxColor = Color.red; // Color of the box
+
+    private RectTransform _rectTransform;
+
+    // Initialize the component
+    private void Awake()
+    {
+        _rectTransform = GetComponent<RectTransform>();
```

Comment:
What if RectTransform component cannot be found? There should be an error here if that happens, explaining exactly why it happened.

Reply:
And if, the current component should be disabled to not print any more errors.

Reply:
If RectTransform is not found, the script now:

- Logs an error explaining the issue.
- Disables itself (enabled = false) to prevent further errors and repeated logging.